### DISCUSSION TITLE
部署规划 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#816] [Announcements]
# 部署规划 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/deploy-plan/

**Comment by timorsea** [comment]
请问worker集群的配置建议是什么样的，谢谢。

**Comment by windsonsea** [comment]
方便加一下社区版交流群吗，群里有很多开发人员，能够快速回答问题

https://docs.daocloud.io/install/intro/#_4
